LoopKitUI/Extensions/Color.swift

```diff
@@ -9,8 +9,8 @@
 import SwiftUI
 
 extension Color {
-    public static var destructive: Color {
-        return .red
-    }
-}
+    public static let destructive = red
 
+    public static let severeWarning = red
+    public static let warning = Color(red: 0.86, green: 0.45, blue: 0)
```

Comment:
[nit] Does this provide a suitable `warning` colour for both light and dark modes?

Reply:
Good catch, will ping design.

Reply:
Updated.